packages/internal/src/__tests__/gql.test.ts

```diff
@@ -0,0 +1,37 @@
+import gql from 'graphql-tag'
```

Comment:
Can we add this as a dev-dependency?

Reply:
Added here: https://github.com/redwoodjs/redwood/pull/2756/files#diff-5babe0fba96044e558791b7fe897b7db1652789839de50f93e34cbe41efc636fR37